test/integration/UniswapV3.t.sol

```diff
@@ -627,11 +629,74 @@ contract UniswapV3IntegrationTests is TestBase {
         assertEq(IERC20(_USDC).balanceOf(_pool), _poolBalanceOfUSDC += bobAmountUSDC_);
     }
 
-    function _createPool() internal returns (address pool_) {
-        pool_ = _factory.createPool(address(_wrappedMToken), _USDC, _POOL_FEE);
+    /* ============ Gas Tests ============ */
+
+    function test_gas_baselinePreparation() public {
+        for (uint256 i_; i_ < 5; ++i_) {
+            address account_ = _accounts[i_];
+
+            _addToList(_EARNERS_LIST, account_);
+            _wrappedMToken.startEarningFor(account_);
+
+            _giveM(account_, 1_000_100e6);
+            _wrap(account_, account_, 1_000_100e6);
+
+            _give(_USDC, account_, 1_000_100e6);
+            _give(_DAI, account_, 1_000_100e6);
+        }
+
+        _addToList(_EARNERS_LIST, _pool);
+        _wrappedMToken.startEarningFor(_pool);
+
+        _setClaimOverrideRecipient(_pool, _carol);
+    }
+
+    function test_uniswapV3_gas_dai_usdc_mint() public {
+        test_gas_baselinePreparation();
+        _pool = _DAI_USDC_POOL;
+        _mintNewPosition(_alice, _DAI, _USDC, _alice, 1_000_000e6);
+    }
+
+    function test_uniswapV3_gas_dai_usdc_swapIn() public {
+        test_uniswapV3_gas_dai_usdc_mint();
+        vm.warp(vm.getBlockTimestamp() + 365 days);
```

Comment:
Why are we wrapping by 365 days? 
Seems useful for the wM/USDC pair to test the gas impact of the rewards calculation but it does not seem to be useful for the DAI/USDC pair.

Reply:
Need to be consistent across the tests to ensure we are comparing apples to apples. The gas snapshot is for the entire test, and cannot differentiate between operations.